evap/student/templates/student_index_semester_evaluations_list.html

```diff
@@ -0,0 +1,168 @@
+{% load evaluation_filters %}
+
+{% for semester in semester_list %}
+    {% if semester.evaluations %}
+        <div class="card{% if not forloop.last %} mb-3{% endif %}{% if semester.results_are_archived %} deactivate{% endif %}">
+            <div class="card-header d-flex">
+                <span class="mr-auto">
+                    {{ semester.semester_name }}
+                </span>
+                {% if semester.results_are_archived %}
+                    <span class="archive-info">
+                        <span class="fas fa-archive"></span> {% trans 'The results of this semester have been archived.' %}
+                    </span>
+                {% endif %}
+            </div>
+            <div class="card-body table-responsive">
+                <table class="table table-seamless-links table-vertically-aligned table-headerless">
+                    <tbody>
+                        {% regroup semester.evaluations by course as course_evaluations %}
+                        {% for course, evaluations in course_evaluations|dictsort:"grouper.name" %}
+                            {% if course.evaluation_count > 1 %}
+                                <tr class="course-row heading-row">
+                                    <td></td>
+                                    <td style="width: 60%">
```

Comment:
Not necessary, but I would prefer to put the `style="width: …"` into a `<colgroup>` to prevent duplicated percentage values (and may get overlooked later)